src/ralph/configuration_management/migrations/0004_delete_scmstatuscheck.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 2.2.28 on 2025-04-14 14:36
+
+from django.db import migrations, transaction
+
+
+def delete_scm_checks(apps, schema_editor):
+    with transaction.atomic():
+        ContentType = apps.get_model("contenttypes", "ContentType")
+        SCMStatusCheck = apps.get_model("configuration_management", "SCMStatusCheck")
+        Permission = apps.get_model("auth", "Permission")
+        scm_content_type = ContentType.objects.filter(app_label='configuration_management', model='scmstatuscheck').first()
+        if scm_content_type:
+            SCMStatusCheck.objects.all().delete()
+            Permission.objects.filter(content_type=scm_content_type).delete()
```

Comment:
It's likely it will fail for prod (there can be related UserPermissions or GroupPermissions). I would suggest removing them manually before deploy and leaving code as it is

Reply:
```
select * from auth_group_permissions where permission_id in (select id from auth_permission where content_type_id = (select id from django_content_type where model ='scmstatuscheck'));
select * from accounts_ralphuser_user_permissions where permission_id in (select id from auth_permission where content_type_id = (select id from django_content_type where model ='scmstatuscheck'));
```